src/components/about-us/section-02/index.js

```diff
@@ -0,0 +1,566 @@
+import { colors } from '../../../themes/common-variables'
+import { foundationIntro, mediaIntro } from '../constants/section-02/org-intro'
+import { gray, numbersInfullPage, numbersInHalfPage, devices } from './utils'
+import { marginBetweenSections } from '../constants/styles'
+import { screen } from '../utils/screen'
+import Arrows from './arrows'
+import categories from '../constants/section-02/categories'
+import categoryIds from '../constants/section-02/category-ids'
+import DepartmentsNameList from './departments-name-list'
+import find from 'lodash/find'
+import foundationMembers from '../constants/section-02/fundation-members'
+import groupBy from 'lodash/groupBy'
+import mail from '../../../../static/asset/about-us/mail.png'
+import mediaMembers from '../constants/section-02/media-members'
+import Navigation from './navigation'
+import PaginatedMemberList from './paginated-list'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import sz from '../constants/screen-size'
+import testImg from '../../../../static/asset/about-us/NicoFang.png'
+import titleImg from '../../../../static/asset/about-us/title-section2.png'
+import titleImgMob from '../../../../static/asset/about-us/title-section2-mob.png'
+
+const _ = {
+  groupBy, find
+}
+
+const halfPageList = [ 2, 3 ] // index of 2, 3 in categoriesAll are rendered in half page
+const members = [ ...mediaMembers, ...foundationMembers ]
```

Comment:
Babel will transpile this to
```
[].concat(mediaMembers, foundationMembers)
```

how about using concat directly, `mediaMembers.concat(foundationMembers)`?